bears/generate_package.py

```diff
@@ -141,6 +197,32 @@ def main():
             if args.upload:
                 perform_upload(os.path.join('bears', 'upload', bear_name))
 
+            if args.conda:
+                substitution_dict['URL'] = (fetch_url(args.conda) or
+                                            ask_question("""
+                                                Repository URL was not found.
```

Comment:
All these spaces in front of Repository will be saved in the string. Is that wanted ?